src/support/README.md

```diff
@@ -24,7 +24,8 @@ feedback and improvements would be very much appreciated.
 Support for CheriBSD is provided via the [CHERI CPU
 Slack](https://www.cl.cam.ac.uk/research/security/ctsrd/cheri/cheri-slack.html),
 where there is a `#cheribsd` channel where many of CheriBSD developers and
-users can be found.
+users can be found and a `#software-porting` channel where discussions on
+software porting to CHERI are held.
```

Comment:
Maybe we should start making this a structured list of some sort, identifying and briefly describing a few key channels?

Reply:
OK.